### ISSUE TITLE
i18n for frontend website.

### ISSUE THREAD

**Issue by mmattbtw** [ScuffleTV/scuffle#53]
### Tell us about your Wish

Add internationalization using something like [svelte-i18n](https://github.com/kaisermann/svelte-i18n) and Crowdin

### Participation

- [X] I am willing to submit a pull request to implement this

### Disclaimer

- [X] I agree

**Comment by TroyKomodo** [comment]
I agree, this can be aimed at a later release it is currently out of the scope for the current release.

**Comment by StarFang208** [comment]
I hope you'll start to open the Crowdin page or something similiar in the future.
